build.psm1

```diff
@@ -31,6 +31,14 @@ if ($IsLinux) {
     $IsUbuntu14 = $IsUbuntu -and $LinuxInfo.VERSION_ID -match '14.04'
     $IsUbuntu16 = $IsUbuntu -and $LinuxInfo.VERSION_ID -match '16.04'
     $IsCentOS = $LinuxInfo.ID -match 'centos' -and $LinuxInfo.VERSION_ID -match '7'
+    $IsFedora = $LinuxInfo.ID -match 'fedora' -and $LinuxInfo.VERSION_ID -ge 24
+    $IsRedHatFamily = $IsCentOS -or $IsFedora
+
+    # Workaround for temporary LD_LIBRARY_PATH hack for Fedora 24
```

Comment:
This is owned by @randomvariable: can you open a PR on top of this to fix up the comments? (Or do it with the documentation additions.)